docs/triage.md

```diff
@@ -15,16 +25,20 @@ just imagine a flowchart
   - e.g. have already discussed not wanting to do or duplicate issue
   - comment acknowledging receipt
   - close
+- do we want to do it, but not in the next year?
+  - comment acknowledging it, but that we don't plan on working on it this year.
+  - label appropriately (examples include `enhancement` or `bug`)
```

Comment:
Can we have a special label for such issues/PRs? Finding them later on, from amongst the closed ones would be a huge task.

Reply:
Also, having a special label for such issues/PRs may look a bit official, that we are respecting their contribution appropriately, and its not just words, that your "Issue/PR will be taken up later"

Reply:
I like that idea. I'll push a commit and add a `future` label.